.github/workflows/release-published.yml

```diff
@@ -0,0 +1,20 @@
+name: Checks after release is published
+on:
+  release:
+    types: ['published']
+
+jobs:
+  validate-github-action:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: k14s/setup-k14s-action@v1
```

Comment:
are we trying to test `setup-k14s-action` or release that we made with this test?

Reply:
we are trying to test that we can retrieve the latest release of `ytt` using the `setup-k14s-action`